helper/src/main/java/me/lucko/helper/menu/Gui.java

```diff
@@ -33,15 +33,12 @@
 import me.lucko.helper.metadata.MetadataMap;
 import me.lucko.helper.terminable.TerminableConsumer;
 import me.lucko.helper.terminable.composite.CompositeTerminable;
-import me.lucko.helper.text.Text;
+import me.lucko.helper.text3.Text;
 import me.lucko.helper.utils.annotation.NonnullByDefault;
 import org.bukkit.Bukkit;
 import org.bukkit.entity.Player;
 import org.bukkit.event.entity.PlayerDeathEvent;
-import org.bukkit.event.inventory.InventoryClickEvent;
-import org.bukkit.event.inventory.InventoryCloseEvent;
-import org.bukkit.event.inventory.InventoryDragEvent;
-import org.bukkit.event.inventory.InventoryOpenEvent;
+import org.bukkit.event.inventory.*;
```

Comment:
Wildcard imports are the bane of OSS.

Reply:
Well I think here it was just intelliJ putting wildcards automatically when there are too much imports of the same package.

Reply:
You can turn that off.

Reply:
It indeed was intelliJ. But i don't really feel the need to turn this off ... and intelliJ does compact this probably with a reason

Reply:
>It indeed was intelliJ. But i don't really feel the need to turn this off ... and intelliJ does compact this probably with a reason

You will find that very few OSS projects (including this one) use wildcard imports. There are very few use cases where it's useful. IntelliJ has it setup by default.

If you create a custom class called "ItemStack", for instance, and then use a wilcard import to reference it and I'm unaware of it, I would _assume_ that you're referring to ItemStack from Bukkit. That's why using wildcard imports is bad for OSS projects, specifically.